### PR TITLE
feat: trash & archive page and sidebar to navigate

### PR DESCRIPTION
**Pull request by pankaj-prog**
trash and archive screens are the same as the notes page, only a few options have changed, and removed new note button 

sidebar : 
sidebar

### CODE REVIEW

**Review by talrejah02 — APPROVED** [review]
- Clean code
- liked the naming conventions
- Nice and clean ui

File: src/components/EditorColumn/EditorColumn.jsx [hunk 1]
```diff
@@ -6,22 +6,40 @@ import ReactQuill from "react-quill";
...
+              <button className="btn btn-alert">Delete Permanently</button>
+            </>
+          )}
+
+          {currentPageName == "archive" && (
+            <>
+              <button className="btn text-btn">Move to trash</button>
+              <button className="btn text-btn">Unarchive</button>
+            </>
+          )}
```
Comment (m26unkwn) [inline src/components/EditorColumn/EditorColumn.jsx:37]: Like that you are writing one component for button.

**Review by m26unkwn — APPROVED** [review]
Everything is good
Clean and modular code.

You can go ahead with it.

File: src/components/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -1,24 +1,51 @@
...
+                    isActive
+                      ? "active-sidebar-link link sidebar-link text-lg"
+                      : "link sidebar-link text-lg"
+                  }
+                  to={link.pathName}
+                >
+                  {link.icon}
+                  <span>{link.name}</span>
+                </NavLink>
+              </li>
```
Comment (gauravsinhaweb) [inline src/components/Sidebar/Sidebar.jsx]: the key is missing here.

**Review by gauravsinhaweb — APPROVED** [review]
- code is well optimized.
- liked the way you have Implemented the method for buttons and icons

**Pull request merged**